javatests/arcs/core/host/HandleAdapterTest.kt

```diff
@@ -33,8 +34,10 @@ import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
+import java.lang.IllegalStateException
```

Comment:
Prefer the kotlin stdlib IllegalStateException instead. (you should be able to just delete this line)